### PR TITLE
Pr auto merge

### PR DESCRIPTION
**Pull request by Madhu-1**
Add mergify rule to auto-merge backported PR

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@nixpanic PTAL

File: .mergify.yml [hunk 1]
```diff
@@ -27,3 +28,18 @@ pull_request_rules:
       backport:
         branches:
           - release-v1.2.0
+  # automerge backports if CI successfully ran
+  - name: automerge backport release-v1.2.0
+    conditions:
+      - author=mergify[bot]
+      - base=release-v1.2.0
+      - label!=DNM
+      - "#changes-requested-reviews-by=0"
```
Comment (nixpanic) [inline .mergify.yml]: Do we not want backports reviewed by someone? Or is it deemed sufficient that a maintainer tags a PR for automatic backporting?

Comment (Madhu-1) [reply .mergify.yml]: as a safer side let me add a check for single review instead of 2 reviews

**Comment by Madhu-1** [comment]
Note: This PR need to be merged manually

**Pull request merged**